README.md

```diff
@@ -25,11 +25,10 @@ For all votes, a simple majority of all *Admin members* for, or against, the iss
 
 ### Contacts for assistance
 - [@ashleygwilliams](https://github.com/ashleygwilliams) - **Ashley Williams**, Individual Membership Director
-- [@bnb](https://github.com/bnb) - **Tierny Cyren**, CommComm Co-Chair
+- [@bnb](https://github.com/bnb) - **Tierny Cyren**, CommComm Chair
```

Comment:
just realized that this should say "Tierney"

Reply:
@oe Good catch 👍 fixed